Shared/Samples/Render multilayer symbols/RenderMultilayerSymbolsView.swift

```diff
@@ -0,0 +1,491 @@
+    private static func makeComplexPolygonGraphic() -> Graphic {
+        // Create a multilayer polygon symbol from the symbol layers.
+        let polygonSymbol = MultilayerPolygonSymbol(
+            symbolLayers: makeLayersForComplexMultilayerSymbols(includeRedFill: true)
+        )
+        
+        // Create a polygon for the graphic.
+        let polygon = Polygon(
+            points: [
+                Point(x: 120, y: 0),
+                Point(x: 140, y: 0),
+                Point(x: 140, y: -10),
+                Point(x: 120, y: -10)
+            ],
+            spatialReference: .wgs84
+        )
+        
+        // Create a graphic of the multilayer polygon symbol.
+        return Graphic(geometry: polygon, symbol: polygonSymbol)
+    }
+    
+    /// Creates a graphic of the complex polyline made with multiple layers.
+    /// - Returns: A new `Graphic` object of the multilayer polyline symbol.
+    private static func makeComplexPolylineGraphic() -> Graphic {
+        // Create a multilayer polyline symbol from the symbol layers.
+        let polylineSymbol = MultilayerPolylineSymbol(
+            symbolLayers: makeLayersForComplexMultilayerSymbols(includeRedFill: false)
+        )
+        
+        // Create a polyline geometry of the graphic.
+        let polyline = Polyline(
+            points: [
+                Point(x: 120, y: -25),
+                Point(x: 140, y: -25)
+            ],
+            spatialReference: .wgs84
+        )
+        
+        // Create a graphic of the multilayer polygon symbol.
+        return Graphic(geometry: polyline, symbol: polylineSymbol)
+    }
+    
+    /// Creates the symbol layers used to create the complex multilayer symbols.
+    /// - Parameter includeRedFill: A Boolean that indicates whether to include a red fill layer.
+    /// - Returns: The new `SymbolLayer` objects.
+    private static func makeLayersForComplexMultilayerSymbols(includeRedFill: Bool) -> [SymbolLayer] {
+        // Create a black dash effect.
+        let dashEffect = DashGeometricEffect(dashTemplate: [5, 3])
+        let blackDashesLayer = SolidStrokeSymbolLayer(width: 1, color: .black, geometricEffects: [dashEffect])
+        blackDashesLayer.capStyle = .square
+        
+        // Create a black outline.
+        let blackOutlineLayer = SolidStrokeSymbolLayer(width: 7, color: .black)
+        blackOutlineLayer.capStyle = .round
+        
+        // Create a yellow stroke.
+        let yellowStrokeLayer = SolidStrokeSymbolLayer(width: 5, color: .yellow)
+        yellowStrokeLayer.capStyle = .round
+        
+        if includeRedFill {
+            // Create a red fill layer.
+            let redFillLayer = SolidFillSymbolLayer(color: .red)
+            return [redFillLayer, blackOutlineLayer, yellowStrokeLayer, blackDashesLayer]
+        } else {
+            return [blackOutlineLayer, yellowStrokeLayer, blackDashesLayer]
+        }
```

Comment:
This could be simplified:
```suggestion
        let symbolLayers = [blackOutlineLayer, yellowStrokeLayer, blackDashesLayer]
        return includeRedFill ? symbolLayers + [SolidFillSymbolLayer(color: .red)] : symbolLayers
```